core/src/main/java/com/axellience/vuegwt/core/client/VueGWT.java

```diff
@@ -75,6 +81,26 @@ public static void initWithoutVueLib()
         for (Runnable onReadyCbk : onReadyCallbacks)
             onReadyCbk.run();
         onReadyCallbacks.clear();
+
+        injectScopedCss();
```

Comment:
This should be called before the onReady callbacks

Reply:
@adrienbaron No, after, because of VueGWT.onReady(() ->VueGWT.registerScopedCss())

Reply:
Oh you're right sorry, didn't remember that it was those 👍